jts-transaction-service/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM fedora:22
+MAINTAINER Gytis Trikleris <gytis@redhat.com>
+WORKDIR /home
+
+ENV NARAYANA_PORT 4710
+ENV NARAYANA_VERSION 5.2.1.Final
+ENV NARAYANA_HOME /home/narayana
+ENV JACORB_LOG_LEVEL 3
```

Comment:
Doesn't seem that one line ENV works well.
I've added this:
```
ENV NARAYANA_VERSION=5.2.1.Final NARAYANA_HOME=/home/narayana NARAYANA_PORT=4710 JACORB_LOG_LEVEL=3
```
And wget tried to get Narayana from this URL:
```
https://www.jboss.org/jbosstm/downloads/5.2.1.Final=NARAYANA_HOME=/home/narayana
```